src/utils/api/transactions.test.js

```diff
@@ -50,6 +50,27 @@ describe('Utils: Transactions API', () => {
       });
     });
 
+    it('should call transactions.get with type', () => {
+      const params = {
+        liskAPIClient,
+        address: '123L',
```

Comment:
I think could not be necessary as it is just one line, but maybe you can use accounts constants and get the address from any account that you need